MC/bin/o2dpg_sim_config.py

```diff
@@ -53,21 +53,52 @@ def add(cfg, flatconfig):
           # specific tunes for high pp
           # run range taken from https://twiki.cern.ch/twiki/bin/viewauth/ALICE/O2DPGMCSamplingSchema
           # taken from JIRA https://alice.its.cern.ch/jira/browse/O2-2691
-          add(config, {"pvertexer.dbscanDeltaT" : 7,
+          # remove extra errors on time margin for tracks and ITS clusters
+          add(config, {"pvertexer.timeMarginTrackTime" : 0.,
+                       "pvertexer.dbscanDeltaT" : 7,
                        "pvertexer.maxChi2TZDebris": 50,
                        "pvertexer.maxMultRatDebris": 1.,
                        "pvertexer.dbscanAdaptCoef" : 20,
                        "pvertexer.maxVerticesPerCluster" : 20,
-                       "pvertexer.dbscanMaxDist2" : 36})
-
+                       "pvertexer.dbscanMaxDist2" : 36,
+                       "ITSCATrackerParam.sysErrY2[0]": 0.,
```

Comment:
This is by default 0, it should not necessary to be set. Did you check that the workflow.json contains this? In my local test it doesn't, and the configuration pointed by Ruben is for apass of data.

Reply:
Indeed you are right, I did not check the default values (that are 0), so they do not need to be set (reverted)